kanopi/kanopi-pack
# Updated dependencies

## Description
> As a package maintainer, I must ensure dependent packages are audited regularly and updated for security and improved functionality.

## Affected URL

*N/A


## Related Tickets

* [29051263](https://kanopi.teamwork.com/app/tasks/29051263)


## Steps to Validate
1. Make sure Kanopi Pack is updated
2. Find a project that uses Kanopi Pack (e.g., FPACT)
3. Update Kanopi Pack in the project to use the updated feature branch
4. Manually test and make sure all functions are working as expected

## Deploy Notes

N/A
